board/pedal/main.c

```diff
@@ -295,6 +297,7 @@ int main() {
   puts("**** INTERRUPTS ON ****\n");
   __enable_irq();
 
+  power_save_init();
```

Comment:
@legonigel is this needed?

Reply:
Nope. This was removed in master, but got left in here on accident.